src/sInv.sol

```diff
@@ -13,12 +13,13 @@ interface IInvEscrow {
 }
```

Comment:
`DBR()` function is non-existent and `distributor()` is unused. Both should be removed.

Reply:
Removed in [c2beee4](https://github.com/InverseFinance/sINV/pull/2/commits/c2beee43668d7d25c5cf5a68f8199752667cc118)